.circleci/config.yml

```diff
@@ -43,45 +59,70 @@ jobs:
             pytest ./Tests/scripts/hook_validations/tests/ -v
             pytest ./Tests/scripts/infrastructure_tests/ -v
             pytest ./Tests/scripts/test_configure_tests.py -v
+      - run:
+          name: Spell Checks
+          when: always
+          command: |
+            python ./Tests/scripts/circleci_spell_checker.py $CIRCLE_BRANCH
+      - run:
+          name: Common Server Documentation
+          when: always
+          command: ./Documentation/commonServerDocs.sh
```

Comment:
Why do we want this in validation_and_testing?

Reply:
It is good to have it. To make sure the common server docs are generated properly. I've had cases that this failed as a result of changing common server.